.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function retry {
```

Comment:
This function is also defined in `.buildkite/scripts/tooling.sh`, could it be imported here?